explorer/client/src/components/Admin/Operators/List.tsx

```diff
@@ -37,7 +39,7 @@ function buildCreatedAtCol(operator: ChainlinkNode): TimeAgoColumn {
 
 function rows(
   operators?: ChainlinkNode[],
-): [TextColumn, UrlColumn, TimeAgoColumn][] | undefined {
+): [UrlColumn, UrlColumn, TimeAgoColumn][] | undefined {
   if (operators) {
     return operators.map(o => {
       return [buildNameCol(o), buildUrlCol(o), buildCreatedAtCol(o)]
```

Comment:
What about optional chaining here instead of wrapping around if?

Reply:
good suggestion, much cleaner :+1: